src/mococrw/hsm.h

```diff
@@ -109,7 +109,7 @@ class HsmEngine : public HSM
     /** Token label used to uniquely identify a token on which objects reside */
     const std::string _tokenLabel;
     /** Pin to access PKCS11 Engine. */
-    const std::string _pin;
+    std::string _pin;
```

Comment:
I am wondering why we keep pin in memory and not just cleanse & forget it after hsm construction?